package/MDAnalysis/coordinates/base.py

```diff
@@ -592,6 +592,49 @@ def volume(self):
         """volume of the unitcell"""
         return core.box_volume(self.dimensions)
 
+    @property
+    def dt(self):
+        """The time difference in ps between timesteps
+
+        .. Note::
+           This defaults to 1.0 ps in the absence of time data
+
+        .. versionadded:: 0.11.0
+        """
+        try:
+            return self.data['dt']
+        except KeyError:
+            warnings.warn("Reader has no dt information, set to 1.0 ps")
```

Comment:
If this only happens once then we should also consider logging the warning, but maybe we leave this to another issue along the lines of which warnings and error messages should be logged and which one just output.